info.plist

```diff
@@ -83,6 +83,8 @@
     /opt/homebrew/bin/php laravel.php "{query}"
 elif [ -f "/usr/local/bin/php" ]; then
     /usr/local/bin/php laravel.php "{query}"
+elif [ -f "$HOME/Library/Application Support/Herd/bin//php" ]; then
+    "$HOME/Library/Application Support/Herd/bin//php" laravel.php "{query}"
```

Comment:
Are the double slashes intentional?

Reply:
That's exactly show up in a "which php"

Reply:
Gotcha. And you tested this locally it works?

Reply:
Yep.  It's what I have setup on my own machine.